qiskit/circuit/__init__.py

```diff
@@ -234,7 +234,6 @@
 from .controlledgate import ControlledGate
 from .instruction import Instruction
 from .instructionset import InstructionSet
-from .operation import Operation
```

Comment:
Do you want to drop this here? This means we're not re-exporting the Operation class from the root `qiskit.circuit`. This will limit who has access to the class. I'm fine with that as we're saying for 0.20.0 we added the class but nothing uses it yet. Just wanted to make sure that was the intent

Reply:
Yes, Matthew, I think it's better this way before we re-add it in way that solves #7528